src/riak_cs_manifest_fsm.erl

```diff
@@ -357,7 +357,6 @@ maybe_backpressure_sleep(Siblings, _BackpressureThreshold) ->
     Delta = MeanSleepMS div 2,
     SleepMS = crypto:rand_uniform(MeanSleepMS - Delta, MeanSleepMS + Delta),
     lager:debug("maybe_backpressure_sleep: Siblings=~p, SleepMS=~p~n", [Siblings, SleepMS]),
-    ok = riak_cs_stats:update(manifest_siblings_bp_sleep, SleepMS * 1000),
```

Comment:
I know this is kludge, bue we have no means whether backpressure sleep happened or not except this stats.
If new stats system has counter, it is sufficient just to count up here.
This PR's aim is to introduce exometer, so It can be deferred to a part of #961.